### PR TITLE
#79: Showing Groups Details in UI

### PR DESCRIPTION
**Pull request by ShashankKasera**

### CODE REVIEW

**Comment by ShashankKasera** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>Having a proper description on PR gives more clarity to the reviewer. Please ensure that the PR has a neatly written description explaining the purpose of this change request.</td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Let's also assign this pull request to someone for review.</td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems">Danger Kotlin</a> against d82b498e09d2dc74d432fed7b92f029207ddfa6a
</p>

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt [hunk 1]
```diff
@@ -30,30 +36,62 @@ class GroupDetailsActivity : AppCompatActivity() {
...
+        cvAddExpenses = findViewById(R.id.cv_add_expenses)
+        tvOverallOweOrOwed = findViewById(R.id.tv_overall_owe)
         llAddGroupMember = findViewById(R.id.ll_group_member)
+        tvOweOrOwedOther = findViewById(R.id.tv_other_member)
+        clOweOwed = findViewById(R.id.cl_amount)
+        tvYouAreTheOnlyOneHere = findViewById(R.id.tv_you_re_the_only_one_here)
+
+        val groupId: Long = intent.extras?.getLong(GROUP_ID) ?: 0
+        val personId = sharedPref.getValue(PERSON_ID, 0L) as Long
+        Log.i("ggu", "onResume: ")
```
Comment (mayankkasera) [inline app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt:82]: Remove unused log.

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt [hunk 2]
```diff
@@ -30,30 +36,62 @@ class GroupDetailsActivity : AppCompatActivity() {
...
+        viewModel.groupDetails(groupId, personId)
     }
 
+    override fun onRestart() {
+        super.onRestart()
+        val groupId: Long = intent.extras?.getLong(GROUP_ID) ?: 0
+        val personId = sharedPref.getValue(PERSON_ID, 0L) as Long
+        Log.i("ggu", "onResume: ")
```
Comment (mayankkasera) [inline app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt:90]: Here as well

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt (hunk 1 above)
Comment (ShashankKasera) [reply app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt]: done

File: app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt (hunk 2 above)
Comment (ShashankKasera) [reply app/src/main/java/com/shashank/splitterexpensemanager/feature/groupdetails/GroupDetailsActivity.kt]: done

**Review by mayankkasera — APPROVED** [review]

**Pull request merged**